src/videotilecontroller/VideoTileController.ts

```diff
@@ -50,12 +52,14 @@ export default interface VideoTileController {
   getLocalVideoTile(): VideoTile | null;
 
   /**
-   * Pauses the video tile, if it exists.
+   * Pauses the video tile if it exists and sends the updated video tile state
```

Comment:
I would suggest adding some explanation here or in the API Overview doc about calling pause on local vs remote tiles.   This API is intended to be called on remote videos.  If called on a local video tile, then the tile will no longer be updated but the local source will continue to be sent into the meeting.

Also include something similar in unpauseVideoTile and the similar API's in VideoTile.ts

Reply:
I have updated it now, thanks!